config/manager/kustomization.yaml

```diff
@@ -1,10 +1,15 @@
 resources:
 - manager.yaml
 
+#images:
```

Comment:
Maybe some quick comments explaining why these are commented out to help future us :grin: